### DISCUSSION TITLE
What is this theme name from the homepage?

### DISCUSSION THREAD

**Discussion by Tsimopak** [codesandbox/codesandbox-client#4869] [General] [answered]
What is the theme name at the homepage?
I couldn't find it

Here's an image:

<img src="https://i.imgur.com/7YmhZYp.png" />

**Comment by garethx** [comment, marked as answer]
It's CodeSandbox Black, which you can get for VS Code here: https://marketplace.visualstudio.com/items?itemName=siddharthkp.codesandbox-black It's also possible to install this in CodeSandbox itself as it differs slightly from the default theme we provide there.
